data/clickprofiledata.js

```diff
@@ -0,0 +1,144 @@
+/*
+Utility script used to generate initial Click Profile data to populate a dataset
+*/
+
+const fs = require('fs');
+const path = require('path');
+const util = require('./util.js');
+
+const LOG_TAG = 'ClickProfile';
+
+const DEFAULT_OUTPUT_FILE = 'clickprofiledata_output.txt';
+
+const DEFAULT_TOTAL_USER_COUNT = 100;
+const DEFAULT_USER_NUMBER_START = 1;
+
+const PREFERENCE_VALUES = [
+  'NONE',
+  'FPS',
+  'ACTION',
+  'RPG',
+];
```

Comment:
Using object here instead of array may make things simpler. Object attributes can be randomly selected too.

Reply:
Thanks, @Pegaiur. I group some of the defaults into their own object. Also, yes object attributes can be randomly selected, but for this situation, I am iterating through these sequentially so I thought a simple array was enough in this situation :)